source/_components/hive.markdown

```diff
@@ -57,6 +62,64 @@ scan_interval:
   default: 2
 {% endconfiguration %}
 
+## Services
+
+### Service `hive.boost_heating`
+
+You can use the service `hive.boost_heating` to set your heating to boost for a period of time at a certain target temperature".
+
+| Service data attribute | Optional | Description                                                             |
+| ---------------------- | -------- | ----------------------------------------------------------------------- |
+| `entity_id`            | no       | String, Name of entity e.g. `climate.heating`.                          |
+| `time_period`          | no       | Time Period, Period of time the boost should last for e.g., `01:30:00`. |
+| `temperature`          | yes      | String, The required target temperature e.g., `20.5`                    |
+
+Examples:
+
+```yaml
+# Example script to boost heating, boost period and target temperature specified.
+script:
+  boost_heating:
+    sequence:
+      - service: hive.boost_heating
+        data:
+          entity_id: "climate.heating"
+          time_period: '01:30:00'
+          temperature: "20.5"
+```
+
+### Service `hive.boost_hot_water`
+
+You can use the service `hive.boost_hot_water` to set your hot water to boost for a period of time.
+
+| Service data attribute | Optional | Description                                                            |
+| ---------------------- | -------- | ---------------------------------------------------------------------- |
+| `entity_id`            | no       | String, Name of entity e.g. `water_heater.hot_water`.                  |
+| `time_period`          | yes      | Time Period, Period of time the boost should last for e.g., `01:30:00`. |
+| `on_off`               | no       | String, The mode to set the boost to on or odd e.g `on`                |
+
+Examples:
+
+```yaml
+# Example script to boost hot water, boost period specified
+script:
+  boost_hot_water:
+    sequence:
+      - service: "hive.boost_hot_water"
+        data:
+          entity_id: "water_heater.hot_water"
+          time_period: '01:30:00'
+          on_off: 'on
+```
+
+## Platforms
```

Comment:
I think we can leave the away

Reply:
Sorry I am not sure what you mean by leave the away?

Reply:
For now it's okay 😉